run/idp-sql/static/firebase.js

```diff
@@ -38,25 +38,6 @@ function signIn() {
 // [END run_end_user_firebase_sign_in]
 // [END cloudrun_end_user_firebase_sign_in]
 
-function signOut() {
```

Comment:
These functions are used in `index.html`

Reply:
💡 Thanks for clarifying. Added them back in.